dapps/assets/src/components/contacts/Contacts.vue

```diff
@@ -0,0 +1,175 @@
+          ></evan-button>
+          <evan-button
+            @click="filterByType('iot-device')"
+            class="filter-btn ml-3"
+            type="text-secondary"
+            icon="mdi mdi-radio-tower"
+            iconPosition="left"
+            :class="{ 'active': filter === 'iot-device' }"
+            :label="$t('_assets.contacts.iot-devices')"
+          ></evan-button>
+          <evan-button
+            @click="resetFilter"
+            class="filter-btn ml-3"
+            type="text-secondary"
+            icon="mdi mdi-account-multiple-outline"
+            iconPosition="left"
+            :class="{ 'active': filter === null }"
+            :label="$t('_assets.contacts.all')"
+          ></evan-button>
+        </div>
+      </div>
+
+      <div class="d-flex flex-row mt-3">
+        <b-table
+          class="evan-table"
+          hover
+          :items="data"
+          :fields="columns"
+          :filter="filter"
+          :filterIncludedFields="filterBy"
+          :tbody-tr-class="'evan-table-body-row'"
+          :thead-tr-class="'evan-table-head-row'"
+          :thead-class="'evan-table-head'"
+          @row-clicked="handleRowClicked"
+          sticky-header="80vh"
+          show-empty
+        >
+          <template
+            v-slot:cell(alias)="data"
+          >{{ data.item.alias ? data.item.alias: data.item.address }}</template>
+          <template v-slot:cell(icon)="data">
+            <i class="table-icon" :class="data.item.icon"></i>
+          </template>
+          <template v-slot:cell(favorite)="data">
+            <i class="table-icon" :class="{'mdi mdi-star': data.item.favorite}"></i>
+          </template>
+
+          <!-- Empty slots -->
+          <template v-slot:empty>
+            <h4>{{ '_assets.contacts.contacts-empty' | translate }}</h4>
+          </template>
+          <template v-slot:emptyfiltered>
+            <h4>{{ '_assets.contacts.filtered-empty' | translate }}</h4>
+          </template>
+        </b-table>
+      </div>
+      <evan-loading v-if="isLoading" :classes="'mt-3'"></evan-loading>
+    </div>
+
+    <evan-button
+      :type="'icon-primary'"
+      size="lg"
+      class="add-contact-btn"
+      icon="mdi mdi-plus"
+      @click="$refs.addContact.showPanel()"
+    ></evan-button>
+
+    <add-contact ref="addContact" @contact-added="handleContactAdded" />
+  </div>
+</template>
+
+<script lang="ts">
+import ContactsComponent from './Contacts';
+export default ContactsComponent;
+</script>
+
+<style lang="scss" scoped>
+@import '~@evan.network/ui/src/style/utils';
+
+.content {
+  max-width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.add-contact-btn {
+  position: fixed;
+  bottom: 40px;
+  right: 60px;
+}
+
+/deep/ .filter-btn.btn {
+  color: cssVar('gray-600');
+  i.left.mdi {
+    margin-right: 0;
+  }
+  &.active {
+    border-bottom: 2px solid cssVar('primary');
+  }
+}
+
+/deep/ .evan-swipe-panel.light {
```

Comment:
Is this needed? Looks good for me on local machine. If yes, should be moved to the core!

Reply:
if needed should be added to the swipe panel. Maybe a theme mixin.

Reply:
Added it here: https://evannetwork.atlassian.net/browse/CORE-794